src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/Main.java

```diff
@@ -8,7 +8,7 @@
 
     private static Map<String, String> urlMangling = new HashMap<String, String>();
     static {
-        urlMangling.put("^", "http://imos-data.aodn.org.au/");
+        urlMangling.put("^", "http://data.aodn.org.au/");
```

Comment:
> Is there a corresponding change required for geoserver config?

Did that already (sorry, didn't want to hold the whole thing, it was complex enough):
https://github.com/aodn/geoserver-config/commit/1b4e9345e2603d1c51c4db78683a12fb52e088dd